siriuspy/siriuspy/oscilloscope/keysight.py

```diff
@@ -183,8 +174,8 @@ def stats_get_data(self):
             data = self.stats_acquire()
             print('Total acquisition time:', _time.time() - tini)
         except Exception:
-            print("Unexpected error:", _sys.exc_info()[0])
-            print('Close connetion by exception')
+            print('Close connection by exception')
+            raise
```

Comment:
esse `raise` sozinho levanta o que?

Reply:
It raises the same Exception, without changes

Reply:
nice, I didn't know that!

But I think the problem with is in the comment below.